### PR TITLE
Fix missing sharing token

### PR DESCRIPTION
**Pull request by vinicius73**

### CODE REVIEW

File: cypress/e2e/share.spec.js [hunk 1]
```diff
@@ -95,16 +95,17 @@ describe('Open test.md in viewer', function() {
 			})
 	})
 
-	it('Shares a folder as a public read only link', function() {
+	it.only('Shares a folder as a public read only link', function() {
```
Comment (max-nextcloud) [inline cypress/e2e/share.spec.js:98]: Pretty sure the only should not be here anymore.

File: cypress/e2e/share.spec.js [hunk 2]
```diff
@@ -95,16 +95,17 @@ describe('Open test.md in viewer', function() {
...
 		cy.shareFile('/folder')
 			.then((token) => {
-				cy.logout()
-				cy.visit(`/s/${token}`)
+				return cy.logout()
+					.then(() => token)
 			})
+			.then((token) => cy.visit(`/s/${token}`))
```
Comment (max-nextcloud) [inline cypress/e2e/share.spec.js:104]: Is this change really needed?
At some point i need to learn how these thenables work in cypress.
I was under the impression that the visit worked fine before.

File: cypress/e2e/share.spec.js [hunk 3]
```diff
@@ -95,16 +95,17 @@ describe('Open test.md in viewer', function() {
...
 			.then(() => {
-				cy.get('#rich-workspace').should('contain', 'Hello world')
+				cy.get('[data-text-el="editor-content-wrapper"]').should('contain', 'Hello world')
 				cy.openFile('test.md')
-				cy.get('#editor-container').should('be.visible')
+				cy.get('[data-text-el="editor-content-wrapper"]').should('be.visible')
 				cy.get('#editor .ProseMirror').should('contain', 'Hello world')
 				cy.get('#editor .ProseMirror h2').should('contain', 'Hello world')
```
Comment (max-nextcloud) [inline cypress/e2e/share.spec.js:110]: I think we should make sure that we are actually looking at the modal and not the workspace here.

**Review by max-nextcloud — COMMENTED** [review]
Thanks a lot for working on this.
The approach looks good. I have some comments on the test.

**Comment by vinicius73** [comment]
> I have some comments on the test.

Thanks!
I've changed the test.

**Review by max-nextcloud — APPROVED** [review]
Thanks for addressing the comments. Looks great now!
